autoscale/src/main/java/com/sequenceiq/periscope/service/ClusterService.java

```diff
@@ -102,7 +102,7 @@ public Cluster create(AutoscaleStackV4Response stack) {
         ClusterPertain clusterPertain =
                 new ClusterPertain(stack.getTenant(), stack.getWorkspaceId(), stack.getUserId(), stack.getUserCrn());
         cluster.setClusterPertain(
-                clusterPertainRepository.findByUserCrn(clusterPertain.getUserCrn())
+                clusterPertainRepository.findByUserId(clusterPertain.getUserId())
```

Comment:
Why change userCRN to userID? userID is an old concept we should rely on userCRN as much as possible.
Also I don't really get the use case why I would want to get a list based on someones CRN.

Reply:
ok updated to use userCRN.